packages/web3-eth/src/web3_eth.ts

```diff
@@ -299,6 +1116,30 @@ export class Web3Eth extends Web3Context<Web3EthExecutionAPI, RegisteredSubscrip
 		return rpcMethodsWrappers.call(this, transaction, blockNumber, returnFormat);
 	}
 
+	/**
+	 * Executes a message call or transaction and returns the amount of the gas used.
```

Comment:
```suggestion
	 *   Generates and returns an estimate of how much gas is necessary to allow the transaction to complete. The transaction will not be added to the blockchain.
```

Reply:
Addressed by [this](https://github.com/ChainSafe/web3.js/pull/5251/commits/1414a1a5ed8f1d0f7737b44df6c92656058cee24) commit